cfe_internal/enterprise/federation/federation.cf

```diff
@@ -81,69 +85,66 @@ bundle agent federation_manage_cfdrop
     "$(create_files)"
       create => "true";
     "$(home)/."
-      depth_search => recurse_with_base("inf"),
-      file_select => all,
-      perms => mo("600", "$(user)");
+      depth_search => default:recurse_with_base("inf"),
+      file_select => default:all,
```

Comment:
default: is needed because of namespacing this bundle? It doesn't degrade/search-up to default if not found in the current namespace? ;) #design #todo?

Reply:
Yes, if you're in a namespace, then it looks in your namespace for the body. If you want to use a body from another namespace, you must do so explicitly.